python/ray/tune/suggest/skopt.py

```diff
@@ -131,7 +149,7 @@ def on_trial_complete(self,
         """
         skopt_trial_info = self._live_trial_mapping.pop(trial_id)
         if result:
-            self._skopt_opt.tell(skopt_trial_info, -result[self._reward_attr])
+            self._skopt_opt.tell(skopt_trial_info, self._metric_op * result[self._metric])
```

Comment:
So `skopt` automatically minimizes, so the metric op actually should be reversed (max => -1). You can leave a comment in the code at L117 for future reference.